docker/README.md

```diff
@@ -258,6 +260,30 @@ You can now start both tunnels:
 ngrok start jetpack jetpack-sftp
 ```
 
+### Docker ngrok
+
+Alternative to the above configuration file is running ngrok in the container with docker-compose file. That starts docker inside a container and you don't have to install it or configure as a standalone software on your machine.
+
+**1. Configure environment**
+
+Add these variables to your `docker/.env` file:
+
+This configures `artpijet.eu.ngrok.io` reserved domain that is available on my basic plan.
+```
+NGROK_AUTH=<your auth key>
+NGROK_SUBDOMAIN=artpijet
+NGROK_REGION=eu
```

Comment:
Should we list what other values here could be and if leaving it empty is ok (I'm assuming it then defaults to US-region)

Reply:
Added explanation in readme.